lib/ms.js

```diff
@@ -17,7 +17,7 @@ var y = d * 365.25;
  *
  * @memberof Mocha
  * @public
- * @api public
+ * @public
```

Comment:
delete this duplicate line

Reply:
For what it's worth, since you're now pulling the "shitcan the @api" task into this PR, to effectively make use of the API documentation, it will need be generated with `--private` cmdline arg to `jsdoc`.

Reply:
whoops, thought I got the dupes.  

- [ ] zap dupes

Reply:
but I don't actually *want* docs about Mocha's private functions on its public API site.  in other words stuff marked `@private` isn't part of Mocha's API.

It's also kind of a way to cover our asses so someone doesn't see something in the docs and think they can or should use it.

Reply:
Private routines are explicitly labeled _private_ in JSDoc output. 
We could always add an `.onclick()` handler to [warn them](https://www.youtube.com/watch?v=RG0ochx16Dg)...
Since API documentation includes the source code, it's not really that hidden to begin with.

But I understand your reasoning...